### ISSUE TITLE
self-harm ideation

### ISSUE THREAD

**Issue by SmartMonkey-git** [obophenotype/human-phenotype-ontology#11708]
**Preferred term label:**
self-harm ideation

**Synonyms**
Self-harm thoughts
Non-suicidal self-injury ideation

**Definition (free text, please give PubMed ID)**
The presence of thoughts of self-harm without the occurrence of self-injurious behaviour.
[PMC12459848](https://pmc.ncbi.nlm.nih.gov/articles/PMC12459848/)

**Parent term (use hpo.jax.org/app)**
Abnormal preoccupation [HP:0025785](https://hpo.jax.org/browse/term/HP:0025785)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0009-0002-1118-4784

**Comment by pnrobinson** [comment]
@SmartMonkey-git 
Adding new term: 
- ID: HP:6001546
- label: Self-harm ideation
- parents: [Abnormal preoccupation (HP:0025785)](https://hpo.jax.org/app/browse/term/HP:0025785)
- pmid: PMID:40991674
- def: Self-harm can be broadly defined as self-injurious behaviour irrespective of motive; self-harm is a broad term that can include non-suicidal self-injury (self-injurious behaviour without intent to die). Self-harm ideation therefore refers to thoughts of self-harm without self-injurious behaviour.
- comment: 
- orcid: orcid:0009-0002-1118-4784
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11708
- synonyms:
